resources/lib/AniListBrowser.py

```diff
@@ -1,11 +1,11 @@
 import itertools
```

Comment:
`Minor pycodestyle fixes` - while i am in favor of using tooling i think that commit alone deserves its own PR.
Also, if using such tool, the configuration file should be shared in the repo (to make sure everyone is aligned), so please add such file to git (tox.ini?)

Reply:
Yeah, was conflicted myself, but those are just a subset of the pycodestyle rules my IDE showed :/ The inner monk wanted the empty lines.

Reply:
Im perfectly fine with your inner monk, it just make sense, i just want the config to be shared so other monks woth get nuts ;)